manager/src/main/java/com/redhat/service/smartevents/manager/api/models/responses/ErrorResponse.java

```diff
@@ -0,0 +1,45 @@
+package com.redhat.service.smartevents.manager.api.models.responses;
+
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class ErrorResponse {
```

Comment:
We already have a `com.redhat.service.smartevents.infra.api.models.responses.ErrorResponse`.

Reply:
I know your class is different; perhaps it'd be better to give it a different name to avoid confusion?

Reply:
`ProcessingError` is a great suggestion from my perspective.

Reply:
Changed all these names to `ProcessingError*`. Thanks for pointing this out :+1: